### DISCUSSION TITLE
Latest 2.0 Release?

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#639] [General]
I know there's no set release window for 2.0, and the beta testing will be a "lengthy" process, and asking for a release date at this point would be pointless, but surely the release of 2.0 couldn't be too far away at this point. 

What this post is about is when could be the latest time we could get 2.0. So far, all I know about 2.0, besides what's listed on the official website, comes from people who have tested the beta version. 

The earliest I could see the release coming is early September, but that's just a guess. To those who have the beta version, when do you think it will be ready for public release?

EDIT: You guys seriously aren't itching for 2.0 to release, especially since in 8 weeks it'll be one year since 1.10 came out? Also, I'm looking for any real beta testers that can share something about 2.0, maybe some footage or screenshots.

**Comment by joelmayer87** [comment]
I‘m a beta tester.

Release will be:

WHEN IT‘S FREAKIN‘ READY, please stop making posts like this every 2 minutes…

Edit: Just saw the OP's edit:

I AM a real beta tester.

I'm looking forward to it but i managed to make a living with animation for the last 10 years before Toonsquid 2.0 so i think i'll be able to hold out for it until it's ready. Again, it's a great update but it won't make animation FOR you.

Sharing Footage or Screenshots of a CLOSED Beta is a really bad idea plus i'm not even sure if it's compliant with the developer's terms of being in the beta programme. With other software that i tested beta versions or even just was SHOWN pre-release stuff i had to sign an NDA so... Lots of things could still change until the final release so i don't think it's in the developers best interest. His blog post basically summarizes everything there is to know about 2.0, right now it's just about usability and stability.

**Comment by trinhpxl** [comment]
of course we all itching for it to release, but theres literally no point to keep asking about it, or pressure anyone whos in beta testing, or the dev- to answer this, lmao. if you already know that asking is pointless.. maybe stop asking ? /lighthearted suggestion.. especially cause so many of us on the github is tired of seeing you being so aggressive with the release date. just wait, like the rest of us ?? 

as for footage or screenshots, im sure its against the common sense to release any of that, considering its a CLOSED beta testing. however, someone / some ppl who do have the beta, are sharing infos in other discussions here about what the newly added features can do. 

like joel said, stop making more discussions about this if all you get is the same answers/replies.. i get it, you're a big supporter of Toonsquid, so am i, but at least show some decency 😭 ?? this is ridiculous that this is the 4th discussion ive seen you post about this. /not starting an argument or anything but like, just being firm on to stop making these discussions. please 😭 🙏
